### ISSUE TITLE
Can I expand one row to Level2 but not another

### ISSUE THREAD

**Issue by hkbeer** [nagarajanchinnasamy/pivottable-subtotal-renderer#50]
In my data source there are 2 columns. I drag them both to left panel as row fields. Say Level1 and Level2 are headers of these 2 columns.

In Level1...items are A, B, C. etc
Level 2 are details. Eg product details...

how to set the code so if items is A only D its Level2 is expanded ...
other are collapsed ie Level2 is not expanded by default?

Thanks

**Comment by nagarajanchinnasamy** [comment]
No. Currently its not possible to collapse by value at a level. You can only simply collapse all the values at a level.
